source/gui/__init__.py

```diff
@@ -524,8 +514,17 @@ def __init__(self, frame: MainFrame):
 
 		self.Bind(wx.adv.EVT_TASKBAR_LEFT_DOWN, self.onActivate)
 		self.Bind(wx.adv.EVT_TASKBAR_RIGHT_DOWN, self.onActivate)
-
+	
+	def evaluateUpdatePendingUpdateMenuItemCommand(self):
+		try:
+			self.menu.Remove(self.installPendingUpdateMenuItem)
+		except Exception:
+			log.debug("Error while removing  pending update menu item", exc_info=True)

```

Comment:
Fix double space in log message

```suggestion
			log.debug("Error while removing pending update menu item", exc_info=True)
```